kotlin/generate-offline-map/src/main/res/layout/dialog_layout.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ProgressBar
+        android:id="@+id/progressBar"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="0dp"
+        android:layout_height="24dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:indeterminate="false"
+        android:progressBackgroundTint="@color/colorPrimary"
```

Comment:
so that we have the progress bar showing blue rather than green, see the export tiles sample for set up, it uses the following here:
```suggestion
        android:theme="@style/progressBarBlue"
```

then place the following in the style.xml file:
```
<!-- Progress bar style-->
<style name="progressBarBlue" parent="@style/Theme.AppCompat">
  <item name="colorAccent">@color/colorPrimary</item>
</style>
```

Reply:
good to know, thanks!